flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/Executor.java

```diff
@@ -133,6 +133,21 @@
 	 */
 	List<String> listUserDefinedFunctions(String sessionId) throws SqlExecutionException;
 
+	/**
+	 * Creates a function with a DDL statement.
+	 */
+	void createFunction(String sessionId, String ddl) throws SqlExecutionException;
```

Comment:
I still think we should introduce `executeSql` method instead of three methods, and i will remove other methods in https://github.com/apache/flink/pull/12188